### PR TITLE
Document-level context via caption

### PR DESCRIPTION
**Pull request by maxkfranz**
- Changes to existing components were kept to a minimum in order to minimise risk.
- The user provides context via free-form text entry into a figure caption text box.
- Article summary credits (contributing author) moved to be alongside the title.
- New metadata field in the model: `caption`.
- Current placeholder text: 'Figure caption (e.g. A phosphorylates B in breast cancer)'.  This example may be improved.

Ref. : Context #506

-----

Default state (empty, placeholder text, not focussed):

Screen Shot 2021-07-21 at 13 20 54

Editing state (with typed text, focussed):

Screen Shot 2021-07-21 at 13 24 22

Article summary:

Screen Shot 2021-07-21 at 13 29 32

Video demo:

https://user-images.githubusercontent.com/989043/126532243-217cea05-2994-4726-bf09-19b75a4f113d.mov

### CODE REVIEW

File: src/client/components/editor/caption.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    const { document } = this.props;
+
+    return h('div.editor-caption', [
+      document.editable() ? 
+        h(TextareaAutosize, {
+          className: makeClassList({
+            'editor-caption-textarea': true
+          }),
+          value: this.state.caption,
+          placeholder: `Figure caption (e.g. A phosphorylates B in breast cancer)`,
```
Comment (maxkfranz) [inline src/client/components/editor/caption.js:24]: This is where the example text lives.  Feel free to directly change it here with an improved placeholder.

Comment (maxkfranz) [reply src/client/components/editor/caption.js:24]: Describe the context of this article (e.g. The set of interactions connecting apoptosis to nuclear function in human breast cancer)

**Comment by jvwong** [comment]
Placeholders ideas:

- `List relevant cell types, physiological conditions, diseases or cellular processes  (e.g. genome stability, cancer, T cell)`
- `List terms describing the context (e.g. cell type, physiological condition, disease, cellular process)`
- `List key cell types, tissues, diseases and cellular/developmental/physiological processes (e.g. neuron, Alzheimer's Disease, translation)`

**Review by jvwong — APPROVED** [review]
Just need a decent  placeholder

**Comment by maxkfranz** [comment]
Added this as a discussion item in the agenda of the next call

**Comment by jvwong** [comment]
`List terms that identify the context (e.g. T cell, cancer, genome stability)`

**Comment by jvwong** [comment]
Distinct-regulation-of-bioenergetics-and-translation-by-group-I-mGluR-and-NMDAR-Biofactoid(1)

Distinct-regulation-of-bioenergetics-and-translation-by-group-I-mGluR-and-NMDAR-Biofactoid

**Comment by gbader** [comment]
Hi - maybe “identify” -> “describe”

**Comment by jvwong** [comment]
Distinct-regulation-of-bioenergetics-and-translation-by-group-I-mGluR-and-NMDAR-Biofactoid(2)

**Comment by maxkfranz** [comment]
@jvwong, the conflict with your other PR has been resolved.  Ready to merge, if the text is finalised

**Pull request merged**